app/uk/gov/hmrc/exports/models/Choice.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.models
+
+case class Choice(value: String)
+
+object Choice {
+  object AllowedChoiceValues {
+    val SupplementaryDec = "SMP"
```

Comment:
Are we able to move this as case'objects extending same trait/abstract class ?

Reply:
Do we need this? It's not required on `ExportsDeclaration`

Reply:
Fixed

Reply:
This is used on the Mappers.